test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -143,7 +144,13 @@ protected virtual void CreateTestStorageEntities()
 
         public virtual void Dispose()
         {
-            Host.Stop();
+            try
+            {
+                Host.Stop();
+            }
+            catch
+            {
+            }
```

Comment:
Yep, that was my thought as well (incomplete host shutdown), but I figured we'd get another nuget merge in before release.

I'll add a test to make sure that the host stops successfully after a listener failure for that 2nd comment.